tool-base/src/main/kotlin/io/spine/tools/code/Language.kt

```diff
@@ -143,21 +152,90 @@ public object CommonLanguages {
      */
     @get:JvmName("any")
     @JvmStatic
-    public val any: Language = object : Language("any language", listOf(""), Glob.any) {
-        override fun comment(line: String): String {
-            throw UnsupportedOperationException("`$name` does not support comments.")
-        }
-    }
+    @Deprecated(
+        "Use `io.spine.tools.code.AnyLanguage` instead.",
+        ReplaceWith("io.spine.tools.code.AnyLanguage")
+    )
+    public val any: Language = AnyLanguage
 
     @get:JvmName("kotlin")
     @JvmStatic
-    public val Kotlin: Language = SlashAsteriskCommentLang("Kotlin", listOf("kt"))
+    @Deprecated(
+        "Use `io.spine.tools.code.Kotlin` instead.",
+        ReplaceWith("io.spine.tools.code.Kotlin")
+    )
+    public val Kotlin: Language = io.spine.tools.code.Kotlin
 
     @get:JvmName("java")
     @JvmStatic
-    public val Java: Language = SlashAsteriskCommentLang("Java", listOf("java"))
+    @Deprecated(
+        "Use `io.spine.tools.code.Java` instead.",
+        ReplaceWith("io.spine.tools.code.Java")
+    )
+    public val Java: Language = io.spine.tools.code.Java
 
     @get:JvmName("javaScript")
     @JvmStatic
-    public val JavaScript: Language = SlashAsteriskCommentLang("JavaScript", listOf("js"))
+    @Deprecated(
+        "Use `io.spine.tools.code.JavaScript` instead.",
+        ReplaceWith("io.spine.tools.code.JavaScript")
+    )
+    public val JavaScript: Language = io.spine.tools.code.JavaScript
+}
+
+/**
+ * This object indicates that any programming language can be accepted.
+ *
+ * Intended to be used for filtering source files by language via file name conventions.
+ * If no filtering required, but a [Language] is needed, use `CommonLanguages.any`.
+ *
+ * Does not support [comments][Language.comment].
+ */
+public object AnyLanguage: Language("any language", listOf(""), Glob.any) {
+    override fun comment(line: String): String {
+        throw UnsupportedOperationException("`$name` does not support comments.")
+    }
+
+    /**
+     * Returns the typed instance of this language for usage in Java code.
+     */
+    @JvmStatic
+    public fun instance(): AnyLanguage = this
```

Comment:
I get why this is `instance()`, not `lang()`. But, I'd still suggest having the same word everywhere for cohesion.

Reply:
The first reason is that `AnyLanguage.lang()` reads worse. The second reason is that this kind of `Language` is not a real one (e.g. like `Kotlin`) but a placeholder for anything.

I'm seriously thinking for renaming `instance()` to`willDo()`.